src/xnnpack/pack.h

```diff
@@ -202,6 +202,38 @@ XNN_INTERNAL void xnn_pack_qs8_qc4w_gemm_goi_w(
   size_t extra_bytes,
   const struct xnn_qs8_qc4w_packing_params* params);
 
+typedef void (*xnn_pack_qs8_qb4w_gemm_fn)(
+  size_t g,
```

Comment:
Traditionally these parameters were undocumented and cryptic. Could you please break this tradition and give them better names?!

Reply:
Updated. I feel nc/kc/nr/kr are quite well "baked" in the code and we should document is somewhere like in the docs dir. It might be more meaningful for readability. I can put up a doc change tomorrow.